CONTRIBUTING.md

```diff
@@ -92,3 +92,5 @@ view, and so forth. This helps.
 4. For minor fixes - open a pull request on GitHub.
 
 Thanks!
+
+[1]: https://github.com/valkey-io/valkey/blob/unstable/SECURITY.md
```

Comment:
Can we use a relative link? Just SECURITY.md

Reply:
Yeah, your suggestion makes more sense.